### PR TITLE
CXXCBC-142:  Update SRV resolution for Windows

### PR DESCRIPTION
**Pull request by thejcfactor**

### CODE REVIEW

**Comment by thejcfactor** [comment]
ugh -- I will try to workout the Windows build.  Wasn't using VS2019 (was using VS2022).

**Review by avsej — APPROVED** [review]

File: core/io/CMakeLists.txt [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+else()
+    target_link_libraries(
+            couchbase_io
+            PRIVATE project_options
+            project_warnings
+            snappy
+            http_parser
+            fmt::fmt
+            spdlog::spdlog)
+endif()
```
Comment (avsej) [inline core/io/CMakeLists.txt:23]: Also looks like you can use multiple `target_link_libraries` calls, and guard only `lphlpapi`.

**Review by avsej — CHANGES_REQUESTED** [review]
Right, code looks good, just failing to build on other windows platform.

File: core/io/CMakeLists.txt (hunk 1 above)
Comment (thejcfactor) [reply core/io/CMakeLists.txt:14]: I didn't know it was additive -- cool!  Updates to guard only `iphlpapi` 👍

**Review by avsej — APPROVED** [review]

**Pull request merged**